### DISCUSSION TITLE
Works with a IMX290-83 IR-CUT Camera

### DISCUSSION THREAD

**Discussion by Counterfeiter** [scriptorron/indi_pylibcamera#4] [General]
Hello,

just wan't to say thanks! Works without any problems on my Pi4 with the IMX290 Color Sensor... This sensor is in a lot of different astro-cams (mostly for planetary and guiding) and works perfectly with this repo!

I have just to update von pycamera2 0.3.6 to 0.3.8... 

best regards
Sebastian

**Comment by scriptorron** [comment]
Hello Sebastian,

Thank you very much for your positive feedback. I am happy to read that it works with your sensor! 
Is your IMX290 connected to the CSI port (flat-band cable) of your Pi4 or is it in a astro-cam and connected with USB?

Best Regards,
Ronald

**Reply by Counterfeiter** [reply]
Hi,

CSI -> https://www.waveshare.com/imx290-83-ir-cut-camera.htm
